### DISCUSSION TITLE
Add support for `.terminal` themes

### DISCUSSION THREAD

**Discussion by leftmove** [ghostty-org/ghostty#4020] [Feature Requests, Ideas] [answered]
A great reason to use Ghostty is all the features it shares with macOS's native terminal. The feature parity is very nice to have.

An optimal solution would be a designated preferences GUI for tinkering with Ghostty's settings (similar to the native terminal) - but I understand this would be a lot to ask for and I am perfectly happy with a simple config file as well.

What I'm asking for in this issue, is more specific. I want support for terminal themes through `.terminal` files. My favorite theme for my IDE, Terminal, and beyond, is [Monokai Pro](https://monokai.pro/).  While Monokai Pro provides a lot of avenues for use, the terminal theme is restricted to a `.terminal` file and a `.itermcolors` file. This means I can't use Monokai Pro for Ghostty (easily at least).

To put it short, I want to use `.terminal` themes for Ghostty.

**Comment by 00-kat** [comment]
I think it's unlikely this would happen, since this would require entirely new parsers for every format people request.  @Kirlovon ported Monokai Pro to Ghostty, though: https://github.com/Kirlovon/monokai-ghostty.

**Reply by 00-kat** [reply]
They PR'd it to the iTerm2 Color Schemes repo, fyi https://github.com/mbadolato/iTerm2-Color-Schemes/pull/486.

**Comment by maman** [comment]
I was able to import my terminal.app colors to https://terminal.sexy/, then export and put the converted color defs manually to my ghostty config. (exporting to xresources seems to be the most readable one)

not easy, but can be done

**Comment by jcollie** [comment, marked as answer]
Ghostty gets all of it's themes from: https://github.com/mbadolato/iTerm2-Color-Schemes
Any themes added to this site are automatically converted to dozens of different formats (including Ghostty). That way Ghostty doesn't need to parse a bunch of different formats. If there's a theme that you'd like included in Ghostty, the way to do it is to add it to the iTerm2-Color-Schemes repo.

**Reply by mitchellh** [reply]
Further: not planned to support any other theme formats.

**Reply by leftmove** [reply]
Thanks - this will do!
